src/DynamoRevit/DynamoRevit.cs

```diff
@@ -546,6 +562,27 @@ private static void TryOpenAndExecuteWorkspaceInCommandData(DynamoRevitCommandDa
                     dynamoViewModel.ShowStartPage = false;
                 }
 
+                //If we have information about the nodes and their values we want to push those values after the file is opened.
+                if (commandData.JournalData.ContainsKey(JournalKeys.ModelNodesInfo))
+                {
+                    try
+                    {
+                        var allNodesInfo = JsonConvert.DeserializeObject<List<Dictionary<string, string>>>(commandData.JournalData[JournalKeys.ModelNodesInfo]);
+                        if (allNodesInfo != null)
+                        {
+                            foreach (var nodeInfo in allNodesInfo)
+                            {
+                                if (nodeInfo.ContainsKey(JournalNodeKeys.Id) && nodeInfo.ContainsKey(JournalNodeKeys.Name) && nodeInfo.ContainsKey(JournalNodeKeys.Value))
+                                {
+                                    DynamoModel.UpdateModelValueCommand modelCommand = new DynamoModel.UpdateModelValueCommand(nodeInfo[JournalNodeKeys.Id], nodeInfo[JournalNodeKeys.Name], nodeInfo[JournalNodeKeys.Value]);
```

Comment:
small comment: can you use `var` here and split this into multiple lines - these are getting a bit too long.